project/CelebornBuild.scala

```diff
@@ -872,3 +882,82 @@ trait FlinkClientProjects {
       )
   }
 }
+
+////////////////////////////////////////////////////////
+//                   MR Client                        //
+////////////////////////////////////////////////////////
+object MRClientProjects {
+
+  def mrClient: Project = {
+    Project("celeborn-client-mr", file("client-mr/mr"))
+      .dependsOn(CelebornCommon.common % "test->test;compile->compile")
+      .dependsOn(CelebornClient.client % "test->test;compile->compile")
```

Comment:
```suggestion
      .dependsOn(CelebornCommon.common, CelebornClient.client)
```

according to `client-mr/mr/pom.xml` we don't need the `test` dependency of project `common` and `client`